assets/js/ajax-filters.js

```diff
@@ -365,6 +367,10 @@ jQuery( document ).ready( function( $ ) {
 					location = $location.val();
 				}
 
+				if( $remote_position.length && $remote_position.is( ':checked' ) ) {
+					remote_position = 'true';
+				}
```

Comment:
This allows only to enable "Remote positions only", but not disable it. For example if the shortcode is `[jobs remote_position=true]` then the "Remote positions only" is checked by default, but unchecking it does not clear the value. It always shows the remote positions only.

Reply:
Ahh good catch, okay so I think this would work instead, as this would set it to null value if the input exists (as in they haven't disabled showing it via a filter) and the checkbox is **unchecked**

```
if( $remote_position.length ) {
    remote_position = $remote_position.is( ':checked' ) ? 'true' : null;
}
``

Passing `null` which will basically override the shortcode "default" value, what do you think?

Reply:
Yes, this should work.

Reply:
@dadish 👍 just pushed the updated to this PR